src/components/Calculator.jsx

```diff
@@ -0,0 +1,48 @@
+import Button from './Button';
+
+const Calculator = () => {
```

Comment:
- [ ] As requested by the previous reviewer,  you should refactor this function should use the `function` keyword to structure this function instead of ES6 arrow function. :pray:

Reply:
I'm getting what you trying to say but what's really the difference between using key word `function` and using `arrow function` ?
 The requirements says use functional components.